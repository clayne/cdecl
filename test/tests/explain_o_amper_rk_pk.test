cdecl @ @ -xc++ @ struct S; explain S* operator&(S&) @ 0
