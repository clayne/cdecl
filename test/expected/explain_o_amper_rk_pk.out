declare & as operator (reference to S) returning pointer to S
